get-function-logs.sh

```diff
@@ -7,16 +7,12 @@ set -u          # Treat unset variables as an error when substituting
 source ./set-project-vars.sh
 
 # Fetch raw logs
-raw_logs=$(gcloud functions logs read "${function_name}" \
-	--region "${region}" \
+raw_logs=$(gcloud logging read "resource.labels.function_name=${function_name}" \
 	--format json \
-	--limit 50 \
-	--sort-by TIME_UTC)
+	--limit 50)
 
-# Format logs
-printf "\n\n"
-echo "${raw_logs}" | jq -r '.[] | if .level == "E" then 
-  "\u001b[31m[\(.level)]\u001b[0m \u001b[33m\(.time_utc)\u001b[0m: \(.log)" 
-else 
-  "[\(.level)] \u001b[33m\(.time_utc)\u001b[0m: \(.log)" 
+echo "${raw_logs}" | jq -r '.[] | if .severity == "ERROR" then
```

Comment:
@chapati23 What do you think of reversing these so that we have the latest log at the bottom of the output? I always find myself having to scroll back up, since those are the ones I'm interested in

Reply:
hmm, in `main` right now (w/o) winston it is actually sorted ASC by timestamp (meaning last logs on the bottom)

maybe winston implicitly has it the other way around or something?

the `--sort-by TIME_UTC` param in the `gcloud functions logs read` command should do exactly that I think

Reply:
oh, i saw you deleted that line, that's the problem :) 
will add it back

Reply:
ah you replaced `gcloud functions logs read` with `gcloud logging read` which doesn't have that flag 🤔

I'll figure it out

Reply:
it's fixed